src/System.Collections.Concurrent/src/System/Collections/Concurrent/PartitionerStatic.cs

```diff
@@ -1492,7 +1493,7 @@ public bool MoveNext()
                 }
             }
 
-            object IEnumerator.Current
+            object? IEnumerator.Current
             {
                 get
                 {
```

Comment:
`override public KeyValuePair<long, TSource> Current` is non-nullable, right?

Reply:
Another explicit implementation case where it doesn't matter.